packages/wonder-blocks-form/src/components/labeled-text-field.js

```diff
@@ -219,7 +225,11 @@ class LabeledTextFieldInternal extends React.Component<
                         field={
                             <TextField
                                 id={`${uniqueId}-field`}
-                                aria-describedby={`${uniqueId}-error`}
+                                aria-describedby={
+                                    ariaDescribedby
+                                        ? ariaDescribedby
```

Comment:
Could you please add a unit test to verify this case? This will help us in the long term to identify any possible errors in case we make any changes.